main.cpp

```diff
@@ -395,8 +412,9 @@ GymChain::~GymChain()
 
 int GymChain::gymChainCounter(int chainLocations, int franchiseLocations)
 {
-    std::cout << "the total number of locations in the chain is: " << chainLocations + franchiseLocations << "\n";
-    return chainLocations + franchiseLocations;
+    this->gymCounter = chainLocations + franchiseLocations;
+    std::cout << "the total number of locations in the chain is: " << gymCounter << "\n";
+    return gymCounter;
 }
 
 double GymChain::profitForecast(int totalGymLocations, int membershipPrice, int totalMembers)
```

Comment:
`profitForecast` is not an action/verb`.
when you create function names, pick names that contain verbs so that the function name describes an action. 
- [ ] revise